nptdms/writer.py

```diff
@@ -30,29 +59,63 @@ def __init__(self, file, mode='w', version=4712):
             It's important that if you are appending segments to an
             existing TDMS file, this matches the existing file version (this can be queried with the
             :py:attr:`~nptdms.TdmsFile.tdms_version` property).
+        :param index_file: Whether or not to write a index file besides the data file. Index files
```

Comment:
Documentation needs updating to reflect the new behaviour of the `index_file` parameter, that it can be a file rather than just a boolean. The defragment function also needs documentation for the `index_file` parameter.